Mathlib/Data/Nat/EvenOddRec.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2022 Stuart Presnell. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Stuart Presnell
 -/
-import Mathlib.Init.Data.Nat.Bitwise
+import Mathlib.Data.Nat.Parity
```

Comment:
I note that `Data.Nat.EvenOddRec` file is a leaf so changing this import is fine.